### PR TITLE
oxidation-of-metals.csl

### PR DESCRIPTION
**Pull request by colinastewart**
Create CSL style for the journal Oxidation of Metals (http://www.springer.com/materials/special+types/journal/11085?detailsPage=pltci_3407509)

### CODE REVIEW

File: oxidation-of-metals.csl [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+    <id>http://www.zotero.org/styles/oxidation-of-metals</id>
+    <link href="http://www.zotero.org/styles/oxidation-of-metals" rel="self"/>
+    <link href="http://www.springer.com/materials/special+types/journal/11085?detailsPage=pltci_3407509" rel="documentation"/>
+    <author>
+      <name>Colin A Stewart</name>
+      <email>colinstewart@engineering.ucsb.edu</email>
+      <!--  This is my first custom CSL style code; I welcome feedback.  -->
+      <!--  Regarding Journal Abbreviations, at this point Zotero doesn't handle them very well, you'll need to check that the Journal Abbr field is populated correctly for each source  -->
+      <!--  Regarding Author Initials, they must be separated by a space, otherwise Zotero will Initiallize eg. Stewart, CA as Stewart, C.  Instead, make sure field lists Stewart, C A with space.  -->
```
Comment (POBrien333) [inline oxidation-of-metals.csl]: CSL can do this for you. use initialize-with=". " in your author and editor macros in the line starting with <name

File: oxidation-of-metals.csl [hunk 2]
```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Oxidation of Metals</title>
+
+    <id>http://www.zotero.org/styles/oxidation-of-metals</id>
+    <link href="http://www.zotero.org/styles/oxidation-of-metals" rel="self"/>
```
Comment (POBrien333) [inline oxidation-of-metals.csl]: Please ad a link to the style you used as a template.

File: oxidation-of-metals.csl [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+      <name>Colin A Stewart</name>
+      <email>colinstewart@engineering.ucsb.edu</email>
+      <!--  This is my first custom CSL style code; I welcome feedback.  -->
+      <!--  Regarding Journal Abbreviations, at this point Zotero doesn't handle them very well, you'll need to check that the Journal Abbr field is populated correctly for each source  -->
+      <!--  Regarding Author Initials, they must be separated by a space, otherwise Zotero will Initiallize eg. Stewart, CA as Stewart, C.  Instead, make sure field lists Stewart, C A with space.  -->
+      <!--  Regarding Web Page citations, I think a few formatting things need to be fixed for journal Oxid Met (italicization)  -->
+    </author>
+    <category citation-format="numeric"/>
+    <category field="science"/>
+    <category field="generic-base"/>
```
Comment (POBrien333) [inline oxidation-of-metals.csl]: remove generic-base ;)

File: oxidation-of-metals.csl [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+      <if variable="volume"/>
+      <else-if variable="DOI">
+        <text variable="DOI" prefix="doi:"/>
+      </else-if>
+      <else-if type="webpage" variable="URL" match="all">
+        <text term="available at" text-case="capitalize-first" suffix=": "/>
+        <text variable="URL" suffix=". "/>
+        <group prefix="" suffix="" delimiter=" ">
+          <text term="accessed" text-case="capitalize-first"/>
+          <date variable="accessed">
```
Comment (POBrien333) [inline oxidation-of-metals.csl]: Replace lines 59-63 with this: <date form="text" variable="accessed"/>

File: oxidation-of-metals.csl [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+  <citation collapse="citation-number">
+    <sort>
+      <key variable="citation-number"/>
+    </sort>
+    <layout prefix="[" suffix="]" delimiter=",">
+      <text variable="citation-number"/>
+    </layout>
+  </citation>
+  <bibliography et-al-min="7" et-al-use-first="3" second-field-align="flush" entry-spacing="0" line-spacing="2">
+    <layout>
```
Comment (POBrien333) [inline oxidation-of-metals.csl]: into layout, add 'suffix="."' so there is a dot at the end of each bibliograpy entry.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**